### PR TITLE
Galician translation - new translation

### PR DESCRIPTION
**Pull request by naciohr**
### Check if this PR fulfills these requirements:
* [X] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [X] I have read the [**CONTRIBUTING**](../../master/Contributing.md) document.
* [X] I made sure there wasn't another [Pull Request](../../../pulls) opened for the same update/change
* [X] I have successfully tested my changes locally

#### Types of changes
- [X] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [ ] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description

Addon translation into Galician - missing translation.

### CODE REVIEW

File: resources/language/resource.language.gl_es/strings.po [hunk 1]
```diff
@@ -0,0 +1,1056 @@
...
+# Addon Provider: libdev + jojo + asciidisco + caphm + CastagnaIT
+msgid ""
+msgstr ""
+"Project-Id-Version: plugin.video.netflix\n"
+"Report-Msgid-Bugs-To: https://github.com/CastagnaIT/plugin.video.netflix\n"
+"POT-Creation-Date: 2020-10-22 11:24+0200\n"
+"PO-Revision-Date: 2020-10-22 11:24+0200\n"
+"Last-Translator: Ignacio Hernández Rial <naciohr@gmail.com>\n"
+"Language-Team: Galego\n"
+"Language: gl\n"
```
Comment (CastagnaIT) [inline resources/language/resource.language.gl_es/strings.po:13]: change to: Language: gl_ES\n

File: resources/language/resource.language.gl_es/strings.po [hunk 2]
```diff
@@ -0,0 +1,1056 @@
...
+msgid "Profile maturity rating for {}"
+msgstr "Valoración do perfil de madurez para {}"
+
+msgctxt "#30233"
+msgid "Only show titles rated [B]{}[/B]."
+msgstr "Mostrar soamente os títulos con valoracións [B]{}[/B]."
+
+msgctxt "#30234"
+msgid "Install Up Next add-on"
+msgstr "Instalar o complemento ‘Proximamente’"
```
Comment (CastagnaIT) [inline resources/language/resource.language.gl_es/strings.po:916]: Here "Up Next" is the name of the add-on, should be not translated

File: resources/language/resource.language.gl_es/strings.po [hunk 3]
```diff
@@ -0,0 +1,1056 @@
...
+msgid "Change watched status (locally)"
+msgstr "Mudar o estado da visualización (localmente)"
+
+msgctxt "#30237"
+msgid "Marked as watched|Marked as unwatched|Restored Netflix watched status"
+msgstr "Marcar como visto|Marcado como non vistor|Restaurado o estado de visualización no Netflix"
+
+msgctxt "#30238"
+msgid "Up Next notifications (watch next video)"
+msgstr "Notificacións ‘Proximamente’ (ver seguinte vídeo)"
```
Comment (CastagnaIT) [inline resources/language/resource.language.gl_es/strings.po:932]: Here "Up Next" is the name of the add-on, should be not translated

File: resources/language/resource.language.gl_es/strings.po [hunk 4]
```diff
@@ -0,0 +1,1056 @@
...
+msgid "Color of supplemental plot info"
+msgstr "Cor da información adicional da sinopse"
+
+msgctxt "#30240"
+msgid "The background services cannot be started due to this problem:[CR]{}"
+msgstr "Os servizos en segundo plano non se puideron iniciar debido a este problema:[CR]{}"
+
+msgctxt "#30241"
+msgid "Open Connect CDN (the lower index is better)"
+msgstr "Abrir conexión CDN (o índice mais baixo é mellor)"
```
Comment (CastagnaIT) [inline resources/language/resource.language.gl_es/strings.po:944]: do not translate "Open Connect" is a corporation name

File: resources/language/resource.language.gl_es/strings.po [hunk 5]
```diff
@@ -0,0 +1,1056 @@
...
+msgstr "Desexa eliminar o estado de visualización de '{}'?[CR][CR]A operación pode demorar até 24h, polo que o título aínda pode aparecer na lista."
+
+msgctxt "#30301"
+msgid "Include Kodi library (only sending data)"
+msgstr "Incluír a biblioteca do Kodi (soamente enviando datos)"
+
+# Unused 30302 to 30339
+msgctxt "#30340"
+msgid "Choose the login method[CR](if the login with E-Mail/Password always returns \"incorrect password\", then use Authentication key, instructions in the GitHub ReadMe)"
+msgstr "Escolla o método de acceso[CR](se o acceso con Correo electrónico/Contrasinal devolve sempre \"contrasinal incorrecto”, entón empregue a chave de autenticación - instrucións no Leme do GitHub)"
```
Comment (CastagnaIT) [inline resources/language/resource.language.gl_es/strings.po:982]: incoherents double accent's

**Review by CastagnaIT — CHANGES_REQUESTED** [review]
only minor things to fix

**Comment by CastagnaIT** [comment]
Thanks very much for the translation

**Comment by naciohr** [comment]
Hi,

All the pointed out elements have been corrected.

Regards,

**Pull request merged**